rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -305,6 +304,23 @@ public static Class build(String fullyQualifiedName,
             }
         }
 
+        /**
+         * This method can be used to find a class by just its qualified type name if it exists in the flyweights map.
+         * If there is more than on Class mapped to the type name, this method returns the first one.
+         *
+         * @param fullyQualifiedName The fully qualified name of the class
+         * @return A class object (if it exists in the flyweights map).
+         */
+        @Nullable public static Class findClass(String fullyQualifiedName) {
+            Set<Class> classes;
+           synchronized (flyweights) {
+               classes = flyweights.get(fullyQualifiedName);
+           }
+           if (classes != null) {
+               return classes.stream().findFirst().orElse(null);
+           }
+           return null;
+        }
```

Comment:
I don't see this used anywhere. What's the use case for this?

Reply:
I pulled these changes out of my "build types on the fly" branch, I do use this method in that branch, I can just remove it for now.

Reply:
I have removed this method.